cdap-gateway/src/main/java/co/cask/cdap/gateway/router/RouterPathLookup.java

```diff
@@ -186,6 +186,11 @@ private String getV3RoutingService(String [] uriParts, AllowedMethod requestMeth
       // /v3/namespaces/{namespace-id}/data/datasets/{name}/properties
       // /v3/namespaces/{namespace-id}/data/datasets/{name}/admin/{method}
       return Constants.Service.DATASET_MANAGER;
+    } else if (uriParts.length >= 2 && uriParts[1].equals("config")) {
+      // TODO: config endpoint only hits stream service
```

Comment:
It's the `StreamHttpService` that runs the `ConfigHandler` -- see https://github.com/caskdata/cdap/blob/develop/cdap-data-fabric/src/main/java/co/cask/cdap/data/stream/service/StreamServiceModule.java#L39: `CommonHandlers.add(handlerBinder);` adds the `ConfigHandler`.